events.php

```diff
@@ -0,0 +1,287 @@
+<?php include 'database.php'; ?>
+<!DOCTYPE html>
+<html>
+<head>
+        <meta charset="utf-8">
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+
+        <meta name="theme-color" content="#D02451" />
+
```

Comment:
@139Vaibhav You need to include the rest of the gymkhana page elements such as navbar, header and footer along with this. Please do that as well.